Modules/posixmodule.c

```diff
@@ -3392,12 +3392,15 @@ os_chown_impl(PyObject *module, path_t *path, uid_t uid, gid_t gid,
      * This is for Mac OS X 10.3, which doesn't have lchown.
```

Comment:
PR #22855  will fix the warnings in posixmodule.c (as a side effect of more interesting changes).